lib/sda/node_test.go

```diff
@@ -0,0 +1,221 @@
+package sda_test
+
+import (
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/cothority/lib/sda"
+	"github.com/satori/go.uuid"
+	"testing"
+	"time"
+)
+
+func TestNodeChannel(t *testing.T) {
+	dbg.TestOutput(testing.Verbose(), 4)
+	names := genLocalhostPeerNames(10, 2000)
+	peerList := genEntityList(tSuite, names)
+	// Generate two example topology
```

Comment:
two example topologies ? or "an example topology (a binary tree)" or just skip the comment